### PR TITLE
added tests for ssgulnur, createFolder and parseJSON functions

### PR DESCRIPTION
**Pull request by mnosov622**
Hi,

I added tests to check the following:

1) Output if bad value is passed to function

2) Output if nothing is passed to a function that expects parameter

3) Match the snapshot

### CODE REVIEW

File: test/unit/options.test.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+const ssgulnur = require('../../src/options');
+
+describe('test the main function with bad values', () => {
+  test('return false if bad argument is passed to a function', () => {
+    expect(ssgulnur('')).toBeFalsy();
+  });
+
+  test('check if calling a function with bad value will match snapshot', () => {
+    expect(ssgulnur('Bad value')).toMatchSnapshot();
```
Comment (gulyapulya) [inline test/unit/options.test.js]: Here, I think it is better to use toBe or toThrow rather than a snapshot as it only returns undefined

**Review by gulyapulya — CHANGES_REQUESTED** [review]
better way possible

File: test/unit/options.test.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+const ssgulnur = require('../../src/options');
+
+describe('test the main function with bad values', () => {
+  test('return false if bad argument is passed to a function', () => {
+    expect(ssgulnur('')).toBeFalsy();
+  });
+
+  test('check if calling a function with bad value will match snapshot', () => {
```
Comment (gulyapulya) [inline test/unit/options.test.js]: sorry but you forgot to change the name

**Review by gulyapulya — CHANGES_REQUESTED** [review]
here little change

File: test/unit/options.test.js (hunk 2 above)
Comment (mnosov622) [reply test/unit/options.test.js]: my bad :)

**Review by gulyapulya — APPROVED** [review]
LGTM!

**Pull request merged**